src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/appointments/AppointmentsToNomisIntTest.kt

```diff
@@ -794,6 +794,38 @@ class AppointmentsToNomisIntTest : SqsIntegrationTestBase() {
     }
   }
 
+  @Nested
+  inner class DeleteAll {
```

Comment:
Not sure we should use this test class - it extends SqsIntegrationTestBase but this test is resource based instead of message based.

Reply:
good idea